src/utilities/introspectionQuery.js

```diff
@@ -33,6 +33,7 @@ export function getIntrospectionQuery(options?: IntrospectionOptions): string {
           args {
             ...InputValue
           }
+          repeatable
```

Comment:
@OlegIlyenko As discussed on the working group we can't assume that legacy servers would support `repeatable`. So this change will break GraphiQL and many other tools.

I would suggest making option `directiveRepeatable` similar to this one:
https://github.com/graphql/graphql-js/blob/f8378d0a6f75b69447e251ad305bc006640d091e/src/utilities/introspectionQuery.js#L12-L16

Long term solution would be to implement two-stage introspection as suggested by Lee.

Reply:
It's a good point, I added an option for it https://github.com/graphql/graphql-js/pull/1541/commits/3b5bf03609b2e92184f511fe7230365628a53182 I also made if disabled by defult in order to maintain backwards compatibility.

Reply:
Also this one: https://github.com/graphql/graphql-js/pull/1541/commits/3a7fafac79e35384c0c40f888c3abc0a6d9d9679 I forgot to update the introspection flow type